packages/limit-service/test/limit-service.test.js

```diff
@@ -85,4 +85,36 @@ describe('Limit Service', function () {
             limitService.isLimited('customThemes').should.be.true();
         });
     });
+
+    describe('Allowlist limit', function () {
```

Comment:
Btw, I've created "limit" specific [test file here](https://github.com/TryGhost/Utils/blob/094b530b299c38b874c3bdea2cee3183e88fd6c4/packages/limit-service/test/limit.test.js). This way we separate test suites per-module straight away (prevents us from having one giant file having all the tests) :wink: